sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,17 +1,16 @@
 # Release History
 
-## 7.10.0-beta.1 (Unreleased)
+## 7.9.2 (2022-07-12)
 
 ### Features Added
 - Exposing `ServiceBusClientBuilder.customEndpointAddress` to support connecting to an intermediary before Azure ServiceBus.
```

Comment:
If this means adding a new public API, then the version update should be a minor version increment and not a patch update.